packages/contractkit/src/wallets/signers/rpc-signer.ts

```diff
@@ -0,0 +1,75 @@
+// @ts-ignore-next-line
+import { account as Account } from 'eth-lib'
+import * as ethUtil from 'ethereumjs-util'
+import { InterceptedMethods } from '../../providers/celo-provider'
+import { RpcCaller } from '../../utils/rpc-caller'
+import { RLPEncodedTx } from '../../utils/signing-utils'
+import { Signer } from './signer'
+
+const decodeSig = (sig: any) => {
+  const [v, r, s] = Account.decodeSignature(sig)
+  return {
+    v: parseInt(v, 16),
+    r: ethUtil.toBuffer(r) as Buffer,
+    s: ethUtil.toBuffer(s) as Buffer,
+  }
+}
+
+const currentTimeInSeconds = () => Math.round(Date.now() / 1000)
+
+export class RpcSigner implements Signer {
+  constructor(
+    protected rpc: RpcCaller,
+    protected account: string,
+    protected unlockBufferSeconds = 5,
```

Comment:
What's the purpose of `unlockBufferSeconds`? Can you add a class header with parameter descriptions?